tests/unit/channel_tests.py

```diff
@@ -392,6 +392,26 @@ def test_basic_qos_raises_channel_closed(self):
         self.assertRaises(exceptions.ChannelClosed, self.obj.basic_qos, 0,
                           False, True)
 
+    def test_basic_qos_raises_prefetch_size_raises_error(self):
```

Comment:
Something doesn't look right with the method name - the word "raises" before "_prefetch_size". Did you mean to name it `test_basic_qos_*invalid*_prefetch_size_raises_error`?

Reply:
Sure did, good catch